src/extension.ts

```diff
@@ -139,6 +141,9 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     },
     () => {
       const msg = (() => {
+        const proxy =
+          `See https://scalameta.org/metals/docs/editors/vscode.html#http-proxy for instructions ` +
```

Comment:
👍 for linking to the website. It's worth to keep in mind that we better not break those urls going forward